installers/ansible/roles/pgo-preflight/tasks/vars.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Include values.yml
+  tags: always
+  block:
+    - name: Check for /conf/values.yaml
+      stat:
+          path: /conf/values.yaml
+      register: conf_dir_result
+
+    - when: conf_dir_result.stat.exists
+      set_fact:
+        config_path: "/conf/values.yaml"
```

Comment:
I like this approach, but I wonder if we should skip checking this directory in the normal ansible case. Something about checking root-path files makes me uneasy.

One of the benefits of the direct inventory file is that `ansible-playbook` accepts it as an argument. 🤔 

It _is_ possible (though a bit cumbersome) to specify a variable on the command line. What do you think of treating this path as a variable? The default could be `{{ playbook_dir }}/values.yaml` and a person/container can override when necessary. I imagine the container with `--extra-vars "something=/conf/values.yaml"`.

Reply:
huh so the pgo-deploy script would have?

```
/usr/bin/env ansible-playbook \
    -i /ansible/inventory.yaml \
    --extra-vars "kubernetes_in_cluster=true" \
    --extra-vars "config_path=/conf/values.yaml" \
    --tags=$DEPLOY_ACTION \
    /ansible/main.yml
```
If we have a default of `config_path={{ playbook_dir }}/values.yaml` we could get rid of that logic. Then we would just have to check that the `{{ config_path }}` exists and include it.

Reply:
I'm not sure this is necessary with only two `extra-vars` but we could have the pgo-deployer specific vars in a file and include it at the command line with `--extra-vars "@/path/to/deployer-specific.yml"`